buoy.php

```diff
@@ -107,13 +107,24 @@ public static function registerL10n () {
      * @return void
      */
     public static function initialize () {
-        // Make sure the WP REST API plugin is installed.
-        // We can remove this if block once that plugin is merged
-        // into WP Core.
-        require_once ABSPATH.'wp-admin/includes/plugin.php';
-        if (is_plugin_inactive('rest-api/plugin.php')) {
-            if (is_wp_error(activate_plugin('rest-api/plugin.php'))) {
-                self::install_plugin_dependency('rest-api');
+        // WordPress versions prior to 4.7 did not fully merge the
+        // WP REST API, on which we (partially) rely. In version 4.7,
+        // however, the REST API's important parts for our use case,
+        // the so-called "Content API" did indeed land in core. This
+        // means we no longer need to force-install it ourselves.
+        //
+        // See https://codex.wordpress.org/Version_4.7 highlights.
+        //
+        // For now, make sure the WP REST API plugin is installed and
+        // activated for users still running older WordPress versions.
+        // We can remove this check once enough Buoys update to v4.7.
+        global $wp_version;
+        if (version_compare('4.7', $wp_version) === 1) {
```

Comment:
Still learning php-things so correct me if I'm misunderstanding this method. For all the `version_compare` calls checking 4.7, shouldn't we be doing a check that passes for >= 4.7 ? i.e. `version_compare('4.7', $wp_version) >= 0`

Even if we change these conditionals upon the next WP release it seems like there would still be bad cases. I'm not familiar with the plugin updating process but if a user installs this version of buoy on their 4.7 wordpress site, and then upgrades wordpress to 4.7+ without upgrading buoy, they will not hit any of these conditionals.